hkpy/hkbase/hkbase.py

```diff
@@ -30,7 +32,7 @@ class HKBase(object):
     """ This class establishes a communication interface with a hkbase.
     """
 
-    def __init__(self, url: str, api_version: str=None, auth: Optional[str]=None):
+    def __init__(self, url: str, api_version: str=None, auth: Optional[str]=None, secret: Optional[str]=None):
```

Comment:
The parameter `secret` is not used in the constructor.

Reply:
Removed